test/fixtures/app/index.html

```diff
@@ -8,6 +8,7 @@
       console.warn('render warn')
       console.error('render error')
     </script>
+    <meta http-equiv="Content-Security-Policy" content="default-src * ; script-src 'self' 'unsafe-inline' ; style-src 'self' ;" />
```

Comment:
This not being here was causing our security warnings to be logged, breaking the logging tests